src/source/Main.scala

```diff
@@ -282,6 +285,15 @@ object Main {
     } else {
       None
     }
+    val objcSwiftBridgingHeaderWriter = if (objcSwiftBridgingHeaderOut.isDefined) {
```

Comment:
I followed the previous implementation.
I tried to the ObjC generator, but without success, so I kept the old implementation.